### PR TITLE
complete mobile porfolio

### PR DESCRIPTION
**Pull request by shobeee**
In this milestone:

- Fixed all linters errors.
- I made use of Flexbox in all 2 sections.
- I sticked to the design as much as possible
- Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+</head>
+<body>
+    <header class = "top">
+        <div class="head">
+           <img src="Logo.svg" class="logo">
```
Comment (levy002) [inline index.html]: - [x] Kindly make sure to wrap your logo into an `a` tag. This is because of three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. :wink:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            <img src="Union.svg" class="union">
```
Comment (levy002) [inline index.html]: - [x] KIndly wrap your hamburger icon into the `nav` tag because of semantics reasons, and the right position for convention and user experience for mobile devices. :wink:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        Glad to see you!</h1>
+        <p class="detail">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p></div>
+        <img src="Connect.svg" class="Connect">
+        <div class="links">
+          <img src="Twitter.svg" class="socialmedia"/>
+          <img src="Linkedin.svg"class="socialmedia"/>
+          <img src="Vector-1.svg"class="socialmedia"/>
+          <img src="github.svg"class="socialmedia"/>
+          <img src="Vector.svg"class="socialmedia"/>
+        </div>
```
Comment (levy002) [inline index.html]: - [x] Kindly wrap your headline container into the `section` tag which represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

Comment (levy002) [inline index.html]: - [x] KIndly make sure to wrap your social media icons into the `ul` tag because of the semantic reason. :wink: 
- [x] Kindly make sure to wrap each of the icons into a `li` tag. :wink: 
- [x] Kindly make sure to warp each icon image in an `a` tag in order to increase the user experience. :wink:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    <header class = "top">
+        <div class="head">
+           <img src="Logo.svg" class="logo">
+            <img src="Union.svg" class="union">
+        </div>
+<div class="paragraph">
+    <h1 class="intro">I'm Shoaib
+        Glad to see you!</h1>
+        <p class="detail">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p></div>
+        <img src="Connect.svg" class="Connect">
```
Comment (levy002) [inline index.html]: - [x] I highly recommend you to use the heading tags (h1,h2,h3, etc..) in order to display the title and sub-title on the website. :wink:

File: index.html (hunk 2 above)
Comment (levy002) [inline index.html]: - [x] Kindly make sure to wrap only the navigation and logo. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. :wink:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @shobeee   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (shobeee) [reply index.html]: @levy002  thank you

File: index.html (hunk 2 above)
Comment (shobeee) [reply index.html]: @levy002 resolved

File: index.html (hunk 3 above)
Comment (shobeee) [reply index.html]: @levy002 thank you it's done

File: index.html (hunk 4 above)
Comment (shobeee) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (shobeee) [reply index.html]: @levy002 thank you i fixed it

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+  transform: translateY(-10px);
+}
+
+.project {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 2rem;
+}
+
+.project-box {
```
Comment (ElsonOtake) [inline styles.css:89]: - I think it's better to align the content of each project or card using Flexbox. It's a project requirement.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+.info span {
+  font-size: 1.5rem;
+  padding: 8px 10px;
+  border-radius: 0.5rem;
+  background: lightgray;
+  color: #6070ff;
+  margin-right: 10px;
+}
+
+.btn {
```
Comment (ElsonOtake) [inline styles.css:136]: - Kindly add the styling for the button pressed as in the template provided. Stick to the template is a project requirement.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+    <p>Back End Dev</p>
+    <img src="Counter.png" alt="">
+    <p>2015</p>
+  </div>
+  <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+  <div class="info">
+    <span>html</span>
+    <span>css</span>
+    <span>javascript</span>
+  </div>
```
Comment (ElsonOtake) [inline index.html]: - I think it's better to use `<ul>` to group the languages and `<li>` for each language. it's a project requirement.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+        <p>Back End Dev</p>
+        <img src="Counter.png" alt="">
+        <p>2015</p>
+      </div>
+      <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="info">
+        <span>html</span>
+        <span>css</span>
+        <span>javascript</span>
+      </div>
```
Comment (ElsonOtake) [inline index.html]: - I think it's better to use `<ul>` to group the languages and `<li>` for each language. it's a project requirement.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+            <p>Back End Dev</p>
+            <img src="Counter.png" alt="">
+            <p>2015</p>
+          </div>
+          <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="info">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
```
Comment (ElsonOtake) [inline index.html]: - I think it's better to use `<ul>` to group the languages and `<li>` for each language. it's a project requirement.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+                <p>Back End Dev</p>
+                <img src="Counter.png" alt="">
+                <p>2015</p>
+              </div>
+              <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="info">
+                <span>html</span>
+                <span>css</span>
+                <span>javascript</span>
+              </div>
```
Comment (ElsonOtake) [inline index.html]: - I think it's better to use `<ul>` to group the languages and `<li>` for each language. it's a project requirement.

File: index.html (hunk 3 above)
Comment (shobeee) [reply index.html]: @levy002 resolved

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,251 @@
...
+
+.info {
+  margin-bottom: 1rem;
+}
+
+.info span {
+  font-size: 1.5rem;
+  padding: 8px 10px;
+  border-radius: 0.5rem;
+  background: lightgray;
```
Comment (ElsonOtake) [inline styles.css:131]: - Please stick to the template provided by Figma. Use the correct background color. Stick to the template is a project requirement.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (ElsonOtake) [inline index.html:3]: - I think it's better to add the `<link>` to fonts used on your portfolio. Without the link, it is not possible to use the fonts in your app. For more informations: [To embed a font, copy the code into the <head> of your html](https://fonts.google.com/specimen/Poppins)

File: index.html [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+          <img src="Twitter.svg" class="socialmedia"/>
+          <img src="Linkedin.svg"class="socialmedia"/>
+          <img src="Vector-1.svg"class="socialmedia"/>
+          <img src="github.svg"class="socialmedia"/>
+          <img src="Vector.svg"class="socialmedia"/>
+        </div>
+    </header>
+
+<section class="project">
+<div class="project-box">
```
Comment (ElsonOtake) [inline index.html:38]: - Kindly use the correct margins and paddings to style the cards according to the template provided. Stick to the template is a project requirement.
Your card.
Screen Shot 2022-09-24 at 10 35 21 PM

Template
Screen Shot 2022-09-24 at 10 35 42 PM

File: index.html [hunk 14]
```diff
@@ -0,0 +1,156 @@
...
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <img src="Connect.svg" class="Connect">
+            <div class="link">
+              <img src="Twitter.svg" class="media"/>
+              <img src="Linkedin.svg"class="media"/>
+              <img src="Vector-1.svg"class="media"/>
+              <img src="github.svg"class="media"/>
+              <img src="Vector.svg"class="media"/>
+            </div>
```
Comment (ElsonOtake) [inline index.html]: - I think it's better to use `<ul>` to list these icons and `<li>` for each icon. It's semantically correct and makes your code more professional.
- Kindly add a link `<a href>` to each icon including your personal address to each of the icons (Twitter, Linkedin, and Github). Your portfolio must present your information to recruiters and hiring managers.

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,251 @@
...
+
+.media {
+  margin-left: 1rem;
+}
+
+.media:hover {
+  transform: translateY(-10px);
+}
+
+.btn-1 {
```
Comment (ElsonOtake) [inline styles.css:210]: - Kindly add the styling for the button pressed as in the template provided. Stick to the template is a project requirement.

File: index.html [hunk 16]
```diff
@@ -0,0 +1,156 @@
...
+                <div class="text-inner">
+                    <h3 class="inner">frameworks</h3>
+                     <i class="fas fa-angle-right"></i>
+                </div>
+                <div class="text-inner">
+                    <h3 class="inner">skills</h3>
+                     <i class="fas fa-angle-right"></i>
+                </div>
+            </div>
+        </div>
```
Comment (ElsonOtake) [inline index.html]: - I think it's better to use `<ul>` and `<li>` for Languages, Frameworks, and Skills. It's semantically correct and is a project requirement.
- Please use `<ul>` and `<li>` for technologies (Javascript, HTML, and CSS). It's semantically correct.
- Kindly create the missing arrow for languages. Stick to the template is a project requirement.
- Kindly correct the position of the arrows for frameworks and skills sticking to the template. Following the template provided is a project requirement.
- Please correct the size of the horizontal line for frameworks and skills according to the template. Stick to the template is a project requirement. 
Your app
Screen Shot 2022-09-24 at 11 04 32 PM

Template
Screen Shot 2022-09-24 at 11 05 04 PM

File: index.html [hunk 17]
```diff
@@ -0,0 +1,156 @@
...
+            <div class="link">
+              <img src="Twitter.svg" class="media"/>
+              <img src="Linkedin.svg"class="media"/>
+              <img src="Vector-1.svg"class="media"/>
+              <img src="github.svg"class="media"/>
+              <img src="Vector.svg"class="media"/>
+            </div>
+            <button class="btn-1" type="button">Get my resume</button>
+        </div>
+        <div class="box">
```
Comment (ElsonOtake) [inline index.html:134]: - I think it's better to use Flexbox to align the content of the "box" instead of just for "tools". Your portfolio must use the same structure for the same group of information.

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @shobeee,

Good job so far! 🎉 🎉 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 18]
```diff
@@ -0,0 +1,164 @@
...
+        <img src="Connect.svg" class="connect">
+        <ul class="links">
+            <li><a href="#"><img src="Twitter.svg" class="socialmedia"/></a></li>
+            <li><a href="#"> <img src="Linkedin.svg"class="socialmedia"/></a></li>
+            <li><a href="#"><img src="Vector-1.svg"class="socialmedia"/></a></li>
+            <li><a href="#"> <img src="github.svg"class="socialmedia"/></a></li>
+            <li><a href="#"> <img src="Vector.svg"class="socialmedia"/></a></li>
+          </ul>
+        </div>
+    </section>
```
Comment (jerryowusu) [inline index.html:34]: ## Note
You submitted the review for the `Portfolio: setup and mobile first`. Since you have made all the required changes by the previous reviewer, your project is going to be approved. :smile:   Kindly note that this approval is for only the `Portfolio: setup and mobile first` and not the `Portfolio: finish mobile version` activity. 
Kindly submit a different pull request for that. :pray:

**Review by jerryowusu — APPROVED** [review]
Hi @shobeee :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

power approve


## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Good commit messages
:heavy_check_mark: Followed GitHub flow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 5 above)
Comment (shobeee) [reply styles.css:89]: noted

File: index.html (hunk 7 above)
Comment (shobeee) [reply index.html]: done

File: index.html (hunk 9 above)
Comment (shobeee) [reply index.html]: thank you for the feedback

File: index.html (hunk 13 above)
Comment (shobeee) [reply index.html:38]: @ElsonOtake fixed

File: index.html (hunk 16 above)
Comment (shobeee) [reply index.html]: done

File: index.html (hunk 17 above)
Comment (shobeee) [reply index.html:134]: @ElsonOtake  thanks

File: index.html (hunk 18 above)
Comment (shobeee) [reply index.html:34]: done

File: styles.css (hunk 6 above)
Comment (shobeee) [reply styles.css:136]: done

File: index.html (hunk 14 above)
Comment (shobeee) [reply index.html]: done

File: styles.css (hunk 15 above)
Comment (shobeee) [reply styles.css:210]: @ElsonOtake thanks

**Review by mgmediaweb — APPROVED** [review]
Hi @shobeee , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
